Classes/shared/ContextManager/SQKContextManager.h

```diff
@@ -73,4 +85,11 @@
  */
 - (NSManagedObjectContext *)newPrivateContext;
 
+/**
+ *  Tear down persistent store backed by coordinator.
+ *
+ *  @return boolean value to indicate success of persistent store tear down.
+ */
+- (BOOL)destroyAndRebuildPersistentStore:(NSError **)error;
```

Comment:
In this comment it would be good to also say that the store is rebuilt after teardown.